code/processes/chainedtp.q

```diff
@@ -38,7 +38,7 @@ openlog:{[lgfile]
     .[set;(lgfile;());{[lgf;err] .lg.e[`openlog;"cannot create new log file : ",string[lgf]," : ", err]}[lgfile]]];
 
   /- backup upd & redefine for counting
-  updold:upd;
+  updold:`. `upd;
```

Comment:
Does it matter if `upd` is undefined and this returns nothing?

Reply:
I think that would be problematic, but not worse than current situation (i.e. that would be a development bug). At this point, upd pretty much must be defined.